content/blog/recipes-1-3-0/index.Rmd

```diff
@@ -0,0 +1,183 @@
+---
+output: hugodown::hugo_document
+
+slug: recipes-1-3-0
+title: recipes 1.3.0
+date: 2025-04-22
+author: Emil Hvitfeldt
+description: >
+    This release brings multiple exciting features and streamlines many recipe steps.
+
+photo:
+  url: https://unsplash.com/photos/background-pattern-3b7sos3CD2c
+  author: James Trenda
+
+# one of: "deep-dive", "learn", "package", "programming", "roundup", or "other"
+categories: [package] 
+tags: [tidymodels, recipes]
+---
+
+```{=html}
+<!--
+TODO:
+* [x] Look over / edit the post's title in the yaml
+* [x] Edit (or delete) the description; note this appears in the Twitter card
+* [x] Pick category and tags (see existing with `hugodown::tidy_show_meta()`)
+* [x] Find photo & update yaml metadata
+* [x] Create `thumbnail-sq.jpg`; height and width should be equal
+* [x] Create `thumbnail-wd.jpg`; width should be >5x height
+* [x] `hugodown::use_tidy_thumbnails()`
+* [x] Add intro sentence, e.g. the standard tagline for the package
+* [x] `usethis::use_tidy_thanks()`
+-->
+```
+
+We're thrilled to announce the release of [recipes](https://recipes.tidymodels.org/) 1.3.0. recipes lets you create a pipeable sequence of feature engineering steps.
+
+You can install it from CRAN with:
+
+```{r, eval = FALSE}
+install.packages("recipes")
+```
+
+This blog post will walk through some of the highlights of this release.
```

Comment:
I would put into words here what awaits the reader. That can help people decide if they want to read it, and when ("oh, I use `step_select()` all the time, I'm gonna read now to be informed"). Without that info, this sentence is very verbose to me and gives me very little info.

Reply:
added more information here https://github.com/tidyverse/tidyverse.org/pull/730/commits/be40d9d626ed8618b3ca9e47bed86274234befbd